Automattic/wp-calypso
# Domains: Reduxify nameservers

#### Changes proposed in this Pull Request

This PR reduxifies the nameservers flux store. Although it looks a bit large, it's mostly a simple port of the flux functionality plus some tests.

#### Preview

Screenshot of the new default error message:
![](https://cldup.com/cJomx-p2DC.png)

#### Testing instructions

* Go to `/domains/manage/:site/name-servers/:domain` and try adding new name servers, editing and deleting existing ones, as well as resetting to the original nameservers and verify everything works well and as it did before.

Fixes #24139.
